connector/build.gradle

```
@@ -6,12 +6,20 @@ plugins {
     id 'maven-publish'
 }
 
-kotlin {
-    jvmToolchain(8)
-}
-
 android {
     compileSdk 34
+
+    compileOptions{
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+        apiVersion = '1.7'
+        languageVersion = '2.0'
```

Shouldn't we set languageVersion only to 1.7 ?
And it is better to use KotlinVersion.KOTLIN_1_7, so we get warned when deprecated

```suggestion
        # languageVersion provides source compatibility with the specified version of Kotlin
        languageVersion = KotlinVersion.KOTLIN_1_7
```

---

Corrected it. Though the KotlinVersion enum would require us to import JetBrains Kotlin DSL.

---

Ok, that's fine with the string then